src/dmd/dmodule.d

```diff
@@ -1001,7 +1001,8 @@ extern (C++) final class Module : Package
         {
             filetype = FileType.c;
 
-            scope p = new CParser!AST(this, buf, cast(bool) docfile, target.c);
+            OutBuffer defines;
```

Comment:
All the work is done in cparse, so I don't see the point of this.

Reply:
cl and sppn do not interleave the #define's into the .i file. It's dumped separately. I did it this way to accommodate both methods.

Reply:
Future PRs will pass in a `defines` that has been initialized with the captured `#defines`.